meteor_mongo_vx.x.x.js

```diff
@@ -53,6 +55,13 @@ declare class Meteor$Mongo$Collection<BaseEntryT> {
       modifier: Meteor$Mongo$Modifier
     ) => boolean
   }): boolean;
+  attachSchema(
+    schema: SimpleSchema,
+    options?: $Shape<{
+      selector: number | string | Meteor$Mongo$ObjectId | Meteor$Mongo$Selector,
+      transform: boolean
+    }>
+  ): void;
```

Comment:
It's also a non-standard method.

Reply:
It’s standard in `aldeed:collection2`. Do you have any idea about how to differentiate standard `meteor/mongo` and `aldeed:collection2` in Flow?

Reply:
That's actually a long-standing issue in Flow: facebook/flow#396. For now, you can either override it locally or create a custom `Meteor` type with the intersection operator:

```js
type Collection<T> = Meteor$Mongo$Collection<T> & {attachSchema(...): ...};
```

Reply:
The problem here is that `new Mongo()` still returns a `Meteor$Mongo$Collection`, so I’m unable to write something like
```
const Posts: Collection<PostProps> = new Mongo('posts');
```

Is there a way to re-declare the `declare module 'meteor/mongo'` part?

Reply:
Nothing I know about. If you declare a new type (anyhow), you can cast (or force cast) your collections to have this type.